.github/workflows/remove-deploy-previews.yml

```diff
@@ -0,0 +1,463 @@
+        with:
+          string: ${{ github.head_ref || github.ref_name }}
+
+  check:
+    # this job will only run if the PR has been merged
+    if: github.event.pull_request.merged == true
+    name: check changed microservice(s)
+    outputs:
+      remove_preview__auth_service: ${{ steps.check_files.outputs.remove_preview__auth_service }} # auth service
+      remove_preview__device_registry: ${{ steps.check_files.outputs.remove_preview__device_registry }} # device registry
+      remove_preview__device_monitoring: ${{ steps.check_files.outputs.remove_preview__device_monitoring }} # device monitoring
+      remove_preview__data_mgt: ${{ steps.check_files.outputs.remove_preview__data_mgt }} # data mgt
+      remove_preview__analytics: ${{ steps.check_files.outputs.remove_preview__analytics }} # analytics
+      remove_preview__device_uptime: ${{ steps.check_files.outputs.remove_preview__device_uptime }} # device uptime
+      remove_preview__device_status: ${{ steps.check_files.outputs.remove_preview__device_status }} # device status
+      remove_preview__locate: ${{ steps.check_files.outputs.remove_preview__locate }} # locate
+      remove_preview__predict: ${{ steps.check_files.outputs.remove_preview__predict }} # predict
+      remove_preview__gp_model: ${{ steps.check_files.outputs.remove_preview__gp_model }} # gp_model
+      remove_preview__exceedances: ${{ steps.check_files.outputs.remove_preview__exceedances }} # exceedances
+      remove_preview__datawarehouse: ${{ steps.check_files.outputs.remove_preview__datawarehouse }} # datawarehouse
+      remove_preview__view: ${{ steps.check_files.outputs.remove_preview__view }} # view
+      remove_preview__calibrate: ${{ steps.check_files.outputs.remove_preview__calibrate }} # calibrate
+      remove_preview__average_jobs: ${{ steps.check_files.outputs.remove_preview__average_jobs }} # average and calibration jobs
+      remove_preview__events_check: ${{ steps.check_files.outputs.remove_preview__events_check }} # event checks
+      remove_preview__notifications: ${{ steps.check_files.outputs.remove_preview__notifications }} # notification
+      remove_preview__kafka_cluster_operator: ${{ steps.check_files.outputs.remove_preview__kafka_cluster_operator }} # kafka cluster operator
+      remove_preview__kafka_cluster: ${{ steps.check_files.outputs.remove_preview__kafka_cluster }} # kafka cluster
+      remove_preview__kafka_topics: ${{ steps.check_files.outputs.remove_preview__kafka_topics }} # kafka topics
+      remove_preview__airflow: ${{ steps.check_files.outputs.remove_preview__airflow }} # airflow
+      remove_preview__network_uptime: ${{ steps.check_files.outputs.remove_preview__network_uptime }} # network_uptime
+      remove_preview__channel_data: ${{ steps.check_files.outputs.remove_preview__channel_data }} # channel data job
+      remove_preview__message_broker_topics: ${{ steps.check_files.outputs.remove_preview__message_broker_topics }} # message broker topics
```

Comment:
Hi @123MwanjeMike. `remove_preview__message_broker_topics` might not be useful

Reply:
Thanks @NoahNsimbe 
Let me address these right away